src/Areas/Extension/Commands/AzqrCommand.cs

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Runtime.InteropServices;
+using AzureMcp.Areas.Extension.Options;
+using AzureMcp.Commands.Subscription;
+using AzureMcp.Services.Azure.Subscription;
+using AzureMcp.Services.ProcessExecution;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Areas.Extension.Commands;
+
+public sealed class AzqrCommand(ILogger<AzqrCommand> logger, int processTimeoutSeconds = 300) : SubscriptionCommand<AzqrOptions>()
+{
+    private const string CommandTitle = "Azure Quick Review CLI Command";
+    private readonly ILogger<AzqrCommand> _logger = logger;
+    private readonly int _processTimeoutSeconds = processTimeoutSeconds;
+    private static string? _cachedAzqrPath;
+
+    public override string Name => "azqr";
+
+    public override string Description =>
+        """
+        Runs Azure Quick Review CLI (azqr) commands to generate compliance/security reports for Azure resources.
+        This tool should be used when the user wants to identify any non-compliant configurations or areas for improvement in their Azure resources.
+        Requires a subscription id and optionally a resource group name. Returns the generated report file's path.
+        Note that Azure Quick Review CLI (azqr) is different from Azure CLI (az).
+        """;
+
+    public override string Title => CommandTitle;
+
+    protected override void RegisterOptions(Command command)
+    {
+        base.RegisterOptions(command);
+        command.AddOption(ExtensionOptionDefinitions.Azqr.OptionalResourceGroup);
+    }
+
+    [McpServerTool(Destructive = false, ReadOnly = true, Title = CommandTitle)]
+    public override async Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        var options = BindOptions(parseResult);
+        var response = context.Response;
+        try
+        {
+            if (!Validate(parseResult.CommandResult, response).IsValid)
+            {
+                return response;
+            }
+
+            ArgumentNullException.ThrowIfNull(options.Subscription);
+
+            var azqrPath = FindAzqrCliPath() ?? throw new FileNotFoundException("Azure Quick Review CLI (azqr) executable not found in PATH. Please ensure azqr is installed.");
```

Comment:
Can we point to any aka.ms link that guides on how to install azqr ?

Reply:
I agree with this.  @conniey has done this for other places.